_locales/en/messages.json

```diff
@@ -137,6 +137,13 @@
   "typingIndicatorsSettingDescription": "See and share when messages are being typed (applies to all sessions).",
   "typingIndicatorsSettingTitle": "Typing Indicators",
   "zoomFactorSettingTitle": "Zoom Factor",
+  "pruneSettingTitle": "Prune Old Open Group Messages",
+  "pruneSettingDescription": "Prune messages older than 6 months from opengroups on start",
+  "pruningOpengroupDialogTitle": "Open group pruning",
+  "pruningOpengroupDialogMessage": "Pruning old open group messages improves performance. Enable pruning for open group messages older than 6 months?",
```

Comment:
Suggest we increase the line height of this text in the confirm modal it looks too squished. Inside `stylesheets/_session.scss` the css class starts at line 554. We should do this
```  
&-main-message {
    font-size: $session-font-md;
    line-height: 1.5;
}
```